internal/util/image/layers.go

```diff
@@ -0,0 +1,99 @@
+package image
+
+import (
+	"archive/tar"
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/containerd/containerd/archive"
+	"github.com/containers/image/v5/pkg/blobinfocache/none"
+	"github.com/containers/image/v5/pkg/compression"
+	"github.com/containers/image/v5/types"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	fsutil "github.com/operator-framework/operator-controller/internal/util/fs"
+)
+
+func ForceOwnershipRWX() archive.Filter {
```

Comment:
Could we doc the funcs ?

Reply:
+1 we should add go docs for these functions.